### ISSUE TITLE
Add more unit tests

### ISSUE THREAD

**Issue by hmol** [hmol/LinkCrawler#3]
This solution needs more unit tests :)

**Comment by paulhtrott** [comment]
Mind if I jump in and help with this? I'm an open source newbie :)

**Comment by hmol** [comment]
Absolutely not, go forth and fork :grinning:

**Comment by paulhtrott** [comment]
Awesome! Thanks! Quick question is there a reason you aren't using a Unit Test project for the Unit Test project? I can fix that also, but don't want to do it if you had a reason for not using it. If fixed it will allow for easier running of the tests in the solution.

**Comment by hmol** [comment]
Just a habit I have to not use the built in Visual Studio Unit test project, don't know why. How is it easier to run tests with this? (Asking because I have not used this before :) )

**Comment by paulhtrott** [comment]
I thought it helped them show up in the test explorer, but it shows, so never mind :smile: I just needed the nUnit Adapter.

**Comment by ghost** [comment]
Is this request processed. I would like to work on this with @paulhtrott if need to. Thanks.

**Comment by paulhtrott** [comment]
No, I gave up on this project. Lots of questions I asked never got answered, so I abandoned my progress. So feel free to take over this request. 👍
